src/core/libraries/videodec/videodec2.cpp

```diff
@@ -0,0 +1,201 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "videodec2.h"
+#include "videodec2_impl.h"
+
+#include "common/assert.h"
+#include "common/logging/log.h"
+#include "core/libraries/error_codes.h"
+#include "core/libraries/libs.h"
+
+namespace Libraries::Vdec2 {
+
+static constexpr u64 kMinimumMemorySize = 32_MB; ///> Fake minimum memory size for querying
+
+int PS4_SYSV_ABI sceVideodec2QueryComputeMemoryInfo(OrbisVideodec2ComputeMemoryInfo* pMemInfoOut) {
+    LOG_INFO(Lib_Vdec2, "called");
+
+    if (!pMemInfoOut) {
+        return ORBIS_VIDEODEC2_ERROR_ARGUMENT_POINTER;
+    }
+    if (pMemInfoOut->thisSize != sizeof(OrbisVideodec2ComputeMemoryInfo)) {
+        return ORBIS_VIDEODEC2_ERROR_STRUCT_SIZE;
+    }
+
+    pMemInfoOut->pCpuGpuMemory = nullptr;
+    pMemInfoOut->cpuGpuMemorySize = kMinimumMemorySize;
+    return ORBIS_OK;
+}
+
+s32 PS4_SYSV_ABI
+sceVideodec2AllocateComputeQueue(const OrbisVideodec2ComputeConfigInfo* pComputeCfgInfoIn,
+                                 const OrbisVideodec2ComputeMemoryInfo* pComputeMemInfoIn,
+                                 OrbisVideodec2ComputeQueue* pComputeQueueOut) {
+    LOG_INFO(Lib_Vdec2, "called");
+    return ORBIS_OK;
+}
+
+s32 PS4_SYSV_ABI sceVideodec2ReleaseComputeQueue(OrbisVideodec2ComputeQueue computeQueueIn) {
+    LOG_INFO(Lib_Vdec2, "called");
+    return ORBIS_OK;
+}
+
+int PS4_SYSV_ABI
+sceVideodec2QueryDecoderMemoryInfo(const OrbisVideodec2DecoderConfigInfo* pCfgInfoIn,
+                                   OrbisVideodec2DecoderMemoryInfo* pMemInfoOut) {
+    LOG_INFO(Lib_Vdec2, "called");
+
+    if (!pCfgInfoIn || !pMemInfoOut) {
+        return ORBIS_VIDEODEC2_ERROR_ARGUMENT_POINTER;
+    }
+    if (pCfgInfoIn->thisSize != sizeof(OrbisVideodec2DecoderConfigInfo) ||
+        pMemInfoOut->thisSize != sizeof(OrbisVideodec2DecoderMemoryInfo)) {
+        return ORBIS_VIDEODEC2_ERROR_STRUCT_SIZE;
+    }
+
+    pMemInfoOut->pCpuMemory = nullptr;
+    pMemInfoOut->pGpuMemory = nullptr;
+    pMemInfoOut->pCpuGpuMemory = nullptr;
+
+    pMemInfoOut->cpuGpuMemorySize = kMinimumMemorySize;
+    pMemInfoOut->cpuMemorySize = kMinimumMemorySize;
+    pMemInfoOut->gpuMemorySize = kMinimumMemorySize;
+
+    pMemInfoOut->maxFrameBufferSize = kMinimumMemorySize;
+    pMemInfoOut->frameBufferAlignment = kMinimumMemorySize;

```

Comment:
Oh, shouldn't this be 0x100 or something?

Reply:
I'm unsure, no clue on what should these values be.

Reply:
Try 0x100 and see if it fixes/breaks anything. This value is used to align framebuffer data for AvPlayer.

Reply:
What do you think should the max framebuffer size be?

Reply:
I think you can leave it as is

Reply:
Got it, I'll try this later.

Reply:
> Try 0x100 and see if it fixes/breaks anything. This value is used to align framebuffer data for AvPlayer.

No change whatsoever.